### PR TITLE
adding backend field for MoU

### PR DESCRIPTION
**Pull request by Rajsnehil**
targets issue #gitmedha/medha-react-ui/issues/432

This PR includes changes related to multiple MoU files upload functionality.

### CODE REVIEW

File: api/institutions/models/institutions.settings.json [hunk 1]
```diff
@@ -82,6 +82,11 @@
     "updated_by_frontend": {
       "plugin": "users-permissions",
       "model": "user"
+    },
+    "mou_list": {
```
Comment (rathorevaibhav) [inline api/institutions/models/institutions.settings.json:86]: update to mou

File: components/common/mou-list.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "collectionName": "components_common_mou_lists",
+  "info": {
+    "name": "mou_list",
+    "icon": "images",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "mou": {
```
Comment (rathorevaibhav) [inline components/common/mou-list.json:10]: the name of the file field would be `mou_file`

**Review by rathorevaibhav — CHANGES_REQUESTED** [review]

File: components/common/mou-list.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+  "collectionName": "components_common_mou_lists",
+  "info": {
+    "name": "mou_list",
```
Comment (rathorevaibhav) [inline components/common/mou-list.json:4]: the name is still mou_list. Have you tested this thoroughly?

File: components/common/mou-list.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
+{
+  "collectionName": "components_common_mou_lists",
```
Comment (rathorevaibhav) [inline components/common/mou-list.json:2]: The collectionName is still mou_lists. Have you self-review and tested this thoroughly?

File: components/common/mou-list.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        "images",
+        "files",
+        "videos"
+      ],
+      "plugin": "upload",
+      "required": false,
+      "pluginOptions": {}
+    },
+    "start_date": {
+      "type": "date"
```
Comment (Rajsnehil) [inline components/common/mou-list.json:23]: added type: date as previously, it was dateTime and was giving the upload time as well.

Comment (rathorevaibhav) [reply components/common/mou-list.json:23]: is there a problem with upload time also? I would actually prefer that if we have time as well

Comment (Rajsnehil) [reply components/common/mou-list.json:23]: there is no problem with that, I just thought that we might only need the date for uploading.

**Comment by Rajsnehil** [comment]
changed date to datetime and merged it.

**Pull request merged**